packages/core-data/src/selectors.js

```diff
@@ -47,31 +48,27 @@ export const isRequestingEmbedPreview = createRegistrySelector(
 /**
  * Returns all available authors.
  *
+ * @deprecated since 11.3. Callers should use `select( 'core' ).getUsers({ who: 'authors' })` instead.
+ *
  * @param {Object}           state Data state.
  * @param {Object|undefined} query Optional object of query parameters to
  *                                 include with request.
  * @return {Array} Authors list.
  */
 export function getAuthors( state, query ) {
+	deprecated( "select( 'core' ).getAuthors()", {
+		since: '11.3',
+		plugin: 'Gutenberg',
```

Comment:
Can we remove the `plugin` key here and update `since` to `5.9` (the idea is that this means WordPress which will help us make better decisions in the future about when to remove this API entirely)